components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2609,6 +2612,12 @@ where
             regions.push(derived.clone());
         }
 
+        let engine_split_result = ctx.engine.batch_split(
+            derived.get_id(),
+            regions.iter().map(|r| r.get_id()).collect(),
+            engine_split_keys.into_iter().collect(),
+        );
```

Comment:
We should hide this operation behind a coprocessor API. E.g., `ctx.host.on_apply_region_split`.